### ISSUE TITLE
{{ROOT_URL}} (for registration) not getting replaced during build

### ISSUE THREAD

**Issue by NullVoxPopuli** [DockYard/ember-service-worker#125]
all I have is this (in my ember-cli-config):
```
    'ember-service-worker': {  },
```
but in the output, I'm getting:

```
(function(){"use strict"
function n(n){e.push(n)}var e=[],r=[]
"serviceWorker"in navigator&&navigator.serviceWorker.register("{{ROOT_URL}}sw.js",{scope:"{{ROOT_URL}}"}).
```

**Comment by NullVoxPopuli** [comment]
this is reproducible running on my host machine: 
![image](https://user-images.githubusercontent.com/199018/44004417-6a638d5a-9e30-11e8-885e-51917b8bedd2.png)

here is my project if you guys ever decide to look https://gitlab.com/NullVoxPopuli/emberclear/tree/master/packages/frontend 

;p

**Comment by bracke** [comment]
I'm also getting this.

**Comment by NullVoxPopuli** [comment]
Started some work on this here: https://github.com/DockYard/ember-service-worker/pull/130

**Comment by stukalin** [comment]
Tried to upgrade the Ember from v3.4 to 3.5 today and received the same issue... Looking forward to see a fix

**Comment by NullVoxPopuli** [comment]
@stukalin how good are you at debugging rollup builds? https://github.com/DockYard/ember-service-worker/pull/130 :D

**Comment by balinterdi** [comment]
I have the same issue but in my case even replacing `{{ROOT_URL}}` with `/' doesn't fix it. The `sw.js` request still has the encoded form prepended to it:

<img width="961" alt="screen shot 2018-10-18 at 16 01 54" src="https://user-images.githubusercontent.com/5022/47160017-4fa48980-d2ef-11e8-837a-4ceb63bd08b1.png">

I also just bumped into this when upgrading to 3.5.

@NullVoxPopuli Unfortunately, I'm very untalented when it comes to debugging rollup builds :(

**Comment by erkie** [comment]
Found it: 

https://github.com/DockYard/ember-service-worker/blob/master/lib/service-worker-builder.js#L88

Needs to be changed to:

`'/**/ember-service-worker/**/*.js'`

Note the leading slash. Then it works!

**Comment by NullVoxPopuli** [comment]
@erkie thanks! in #132, I've made your change and it fixed my issue in emberclear.io .

**Comment by eshtadc** [comment]
Fixed in #132
